lnrpc/rpc.proto

```diff
@@ -337,12 +337,15 @@ service Lightning {
     }
 
     /* lncli: `sendtoroute`
-    SendToRoute is a bi-directional streaming RPC for sending payment through
-    the Lightning Network. This method differs from SendPayment in that it
-    allows users to specify a full route manually. This can be used for things
-    like rebalancing, and atomic swaps.
+    Deprecated, use routerrpc.SendToRouteV2. SendToRoute is a bi-directional
+    streaming RPC for sending payment through the Lightning Network. This
+    method differs from SendPayment in that it allows users to specify a full
+    route manually. This can be used for things like rebalancing, and atomic
+    swaps.
     */
-    rpc SendToRoute (stream SendToRouteRequest) returns (stream SendResponse);
+    rpc SendToRoute (stream SendToRouteRequest) returns (stream SendResponse) {
```

Comment:
perhaps we should also log via WRN whenever a call over the deprecated RPCs is made? may be a good time to do so for previously deprecated payment RPCs

Reply:
I like the idea, but looks fine to me to do in another PR. It could be that there are different opinions on this.